cmake/setup.sh.in

```diff
@@ -1,7 +1,22 @@
+# if GAZEBO_INSTALL_PREFIX is unset, fall back to
+# configure-time value of CMAKE_INSTALL_PREFIX
+installPrefix=${GAZEBO_INSTALL_PREFIX:-@CMAKE_INSTALL_PREFIX@}
+if [ ! -d "${installPrefix}" ]; then
+  echo "The install prefix ${installPrefix} does not exist."
+  unset installPrefix
+  return 1
+elif [ ! -f "${installPrefix}/@GAZEBO_RELATIVE_RESOURCE_PATH@/setup.sh" ]; then
```

Comment:
Is this checking for the existing of the setup.sh produced by this file? Under what circumstance might this condition be true?

Reply:
This is checking that the path in the `installPrefix` variable contains at least one expected file. In this case it resolves to `${installPrefix}/share/gazebo-11/setup.sh` for gazebo11, but I suppose we could equivalently choose `${installPrefix}/share/gazebo-11/worlds/empty.world`. The first `if` statement checks that the `$installPrefix` path exists; the second checks for some expected content.

I suppose it's possible to confuse matters if you manually set `GAZEBO_INSTALL_PREFIX` to one install path but source the `setup.sh` from a different install path, but the recommended approach is now to use `setup.bash`, which should avoid that type of confusion